reporter/src/funTest/kotlin/reporters/AsciidocTemplateReporterFunTest.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.reporter.reporters
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+import java.io.File
+
+import kotlin.IllegalArgumentException
+
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.reporter.ORT_RESULT
+import org.ossreviewtoolkit.reporter.ReporterInput
+import org.ossreviewtoolkit.utils.ORT_NAME
+
+class AsciidocTemplateReporterFunTest : StringSpec({
+    "PDF output is created successfully from an existing result and default template" {
+        val report = generateReport(ORT_RESULT)
+
+        report.single().length() shouldBe 84514L
+    }
+
+    "Report generation is aborted when path to non-existing pdf-them file is given" {
+        shouldThrow<IllegalArgumentException> {
+            generateReport(ORT_RESULT, mapOf("pdf-theme.path" to "dummy.file"))
+        }
+    }
+})
+
+private fun generateReport(
+    ortResult: OrtResult,
+    options: Map<String, String> = emptyMap()
+): List<File> {
+    val input = ReporterInput(ortResult)
+
+    val outputDir = createTempDir(
+        ORT_NAME,
+        AsciidocTemplateReporterFunTest::class.simpleName
+    ).apply { deleteOnExit() }
```

Comment:
Now needs to be replaced with

    val outputDir = createTempDirectory("$ORT_NAME-${AsciidocTemplateReporterFunTest::class.simpleName}").toFile()
        .apply { deleteOnExit() }

and import of

    import kotlin.io.path.createTempDirectory

(Note the import order! :wink: )